addons/web/static/src/js/views/kanban/kanban_column.js

```diff
@@ -82,10 +82,12 @@ var KanbanColumn = Widget.extend({
 
         if (options.group_by_tooltip) {
             this.tooltipInfo = _.map(options.group_by_tooltip, function (help, field) {
-                return (data.tooltipData && data.tooltipData[field] && "<div>" + help + "<br>" + data.tooltipData[field] + "</div>") || '';
+                if (data.tooltipData && data.tooltipData[field]) {
+                    help = help ? help + "</br>" : help;
+                    data.tooltipData[field] = (!help && !_.values(data.tooltipData).includes(false)) ? data.tooltipData[field] + "<div class='dropdown-divider' role='separator' />" : data.tooltipData[field];
```

Comment:
includes is ES6 (not supported by ie11), so you can't use it. You can use indexOf instead.
Also, this line is way too long and hard to read/understand, i would try to re-write it to make it more readable.

Reply:
@aab-odoo It seems that something went wrong when @tivisse did rebase, I removed ```includes``` when reviewed it, I rebased branch again, you can check it now, I will make separate commit for web part, Thanks